ansible-scylla-monitoring/defaults/main.yml

```diff
@@ -34,6 +35,7 @@ scylla_monitoring_dashboards_versions:
 # If "default" is set, the default file will be taken from the prometheus directory in the unpacked monitoring stack distribution.
 #
 # See the original default file at https://github.com/scylladb/scylla-monitoring/blob/master/prometheus/prom_rules/prometheus.rules.yml
+# above file will STILL be used, below file if other than default will be added to above file as addition (along to all other files inside prom_rules)
```

Comment:
I struggle to understand the sentence, read it three times :(

Reply:
rewritten and amended